app/(request)/private-cloud/decision/[licencePlate]/page.tsx

```diff
@@ -128,6 +144,14 @@ export default function RequestDecision({ params }: { params: { licencePlate: st
         handleSubmit={methods.handleSubmit(onSubmit)}
         isLoading={isLoading}
       />
+      <Comment
+        open={openComment}
+        setOpen={setOpenComment}
+        handleSubmit={methods.handleSubmit(onSubmit)}
+        isLoading={isLoading}
+        onCommentChange={handleComment}
```

Comment:
1. Let's remove `onCommentChange` along with `handleComment` & `setComment`. etc..
2. rename `handleSubmit` to `onSubmit` to take a callback function that takes `comment` value as the only parameter.
3. and then set the `comment` value with `methods.setValue` method.